analyzer/consensus/registration.go

```diff
@@ -37,3 +38,11 @@ func registerRelatedOCAddress(relatedAddresses map[apiTypes.Address]struct{}, oc
 	relatedAddresses[addr] = struct{}{}
 	return addr, nil
 }
+
+func stringifyCmtAddress(cmtAddr cmtTypes.Address) (apiTypes.Address, error) {
+	var sdkAddr sdkTypes.Address
+	if err := sdkAddr.UnmarshalBinary(cmtAddr); err != nil {
```

Comment:
this is wrong, it's not the same kind of address.

see oasis's cometbft address impl. need to make a lookup

Reply:
ok after a lot of fuss, the lookup is made